realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -99,14 +99,14 @@ private void populateTestRealm(Realm testRealm, int dataSize) {
             allTypes.setColumnBinary(new byte[]{1, 2, 3});
             allTypes.setColumnDate(new Date(DECADE_MILLIS * (i - (dataSize / 2))));
```

Comment:
That is actually a great argument. I think right now we are fairly inconsistent, but I'm all for enforcing uppercasing the type suffix 👍